EXERCISES/exercise10.js

```diff
@@ -0,0 +1,18 @@
+/**​Crie uma função que verifica se um número inteiro passado como parêmetro é divisível por 3 e retorne true
+ou false. */
+
+function divisivelPor3 (num) {
```

Comment:
sei que o enunciado pede por 3, mas o que acha de tornar isso mais dinâmico?

```suggestion
function divisivelPorX (num, X) {
```